pkg/scheduler/plugins/capacity/capacity_test.go

```diff
@@ -654,6 +654,205 @@ func Test_capacityPlugin_OnSessionOpenWithHierarchy(t *testing.T) {
 	}
 }
 
+func TestOverwriteRootQueueRealCapability(t *testing.T) {
```

Comment:
I think we can just add a simple case in `Test_capacityPlugin_OnSessionOpenWithHierarchy` is enough, there's no need to add a new `Test_capacityPlugin_OnSessionOpenWithHierarchy` test func and add a lot of new cases :)